package oop.exception;

public class Exception_07 {

	public static void main(String[] args) {
		
		int num1 = 100, num2 = 0;
		int res = num1/ num2;
		System.out.println("결과 : " + res);
	}

}
